aspnetcore/host-and-deploy/windows-service.md

```diff
@@ -229,7 +229,13 @@ Services that interact with requests from the Internet or a corporate network an
 
 ## Configure HTTPS
 
-Specify a [Kestrel server HTTPS endpoint configuration](xref:fundamentals/servers/kestrel#endpoint-configuration).
+To configure the service with a secure endpoint:
+
+1. Create an X.509 certificate.
+1. Specify a [Kestrel server HTTPS endpoint configuration](xref:fundamentals/servers/kestrel#endpoint-configuration) to use the certificate.
+1. Deploy the certificate with the app.
+
+Use of the ASP.NET Core HTTPS development certificate to secure a service endpoint isn't supported. The developer certificate is stored in the user's Personal certificate store, but the service runs using the [LocalSystem account](/windows/desktop/services/localsystem-account) and has its own certificate store. We recommend creating a certificate and deploying it with the app.
```

Comment:
We do? That seems odd. I'd create the cert on the machine it's going to be on, so the private key isn't ever on a developer's machine. Also Local System? Local System is high privilege. Shouldn't we be recommending creating a specific user here?

Reply:
@blowdart I was following @Tratcher's lead: https://github.com/aspnet/Hosting/issues/1573#issuecomment-432112672

... sure thing ... just let me know what it should say, and I'll wordsmith it.

Reply:
@blowdart create the cert any way you like. The point here is that you have to do it yourself.

Reply:
@Tratcher what about local system? That can't be a real recommendation here.

Reply:
@guardrex Something like "We recommend using your platform's certificate acquisition and deployment mechanisms to ensure a certificate is available on the host machine"?

Reply:
What account would you recommend?

Reply:
We're talking about a broader update than just the cert now. After I hear back on what you want to do about the user account for the service, I'll open a new issue and address that quickly (probably today).

See if the last commit :point_down: tides this over until the next PR.

Reply:
A custom one created for the app. Not a built in account at all.

Reply:
> A custom one created for the app. Not a built in account at all.

I've moved this piece to https://github.com/aspnet/Docs/issues/9226, and I'll perform these updates very soon ... within the next few days I think.